tests/integration/repos_test.go

```diff
@@ -8,11 +8,10 @@
 package tests
 
 import (
+	"github.com/google/go-github/github"
 	"net/http"
 	"reflect"
 	"testing"
-
-	"github.com/google/go-github/github"
```

Comment:
No reason to change the above. The previous version was compatible with `goimports`, the new one isn't.